### DISCUSSION TITLE
DataGrid Groups default to collapsed

### DISCUSSION THREAD

**Discussion by ploring04** [AvaloniaUI/Avalonia#9936] [Q&A]
With Avalonia 11.0.0-Preview-4 I think I am missing something.  

I have a simple datagrid with a group (Person Name is the group and then just some orders) and I have been trying all sorts of things to get all of the DataGridRowGroupHeaders to be collapsed upon initial viewing and only when selected expand the group so there would be at most 1 group expanded at any given time.  The DataGrid property RowDetailsVisibilityMode only seems to work with RowDetails and not DataGridRowGroupHeaders.  I have also tried using the CollapseRowGroup but not sure how to make the first parameter work as it gives me an index out of range exception.

**Comment by JohnnyP0T** [comment]
Did you find a solution ?

**Comment by cornerbowlsoftware** [comment]
I'm looking for this solution as well.  Does anyone know?  Looking at the source code it appears to be impossible?

```
        internal void EnsureExpanderButtonIsChecked()
        {
            if (_expanderButton != null && RowGroupInfo != null && RowGroupInfo.CollectionViewGroup != null &&
                RowGroupInfo.CollectionViewGroup.ItemCount != 0)
            {
                SetIsCheckedNoCallBack(RowGroupInfo.IsVisible);
            }
        }
```
